classes/migration/upgrade/PKPv3_3_0UpgradeMigration.inc.php

```diff
@@ -137,8 +137,12 @@ public function up() {
 		$this->_fixCapitalCustomBlockTitles();
 		$this->_createCustomBlockTitles();
 
+		// Remove item views related to submission files and notes,
+		// and convert the assoc_id to an integer
+		Capsule::table('item_views')
+			->where('assoc_type', '!=', ASSOC_TYPE_REVIEW_RESPONSE)
+			->delete();
 		Capsule::schema()->create('item_views', function (Blueprint $table) {
-			// Now that submission file IDs are not compound, convert item_views.assoc_id to single ID
 			$table->bigInteger('assoc_id')->change();
```

Comment:
Note that string to int casts in PostgreSQL require an explicit cast; fixed in this commit: https://github.com/pkp/pkp-lib/commit/a9a5499a0b4e50baa435b7ecb4c3c9ecee04e19f#diff-f4951b8d90b205bf60a1111a761b62eb75f598c0c536738da01d027561a33f1cR146

Reply:
Thanks!